scripts/sct_image.py

```diff
@@ -31,169 +30,191 @@ def get_parser():
-    parser.add_option(name="-getorient",
-                      description='Get orientation of the input image',
-                      mandatory=False)
-    parser.add_option(name="-setorient",
-                      type_value="multiple_choice",
-                      description='Set orientation of the input image (only modifies the header).',
-                      mandatory=False,
-                      example='RIP LIP RSP LSP RIA LIA RSA LSA IRP ILP SRP SLP IRA ILA SRA SLA RPI LPI RAI LAI RPS LPS RAS LAS PRI PLI ARI ALI PRS PLS ARS ALS IPR SPR IAR SAR IPL SPL IAL SAL PIR PSR AIR ASR PIL PSL AIL ASL'.split())
-    parser.add_option(name="-setorient-data",
-                      type_value="multiple_choice",
-                      description='Set orientation of the input image\'s data (does NOT modify the header, but the data). Use with care !',
-                      mandatory=False,
-                      example='RIP LIP RSP LSP RIA LIA RSA LSA IRP ILP SRP SLP IRA ILA SRA SLA RPI LPI RAI LAI RPS LPS RAS LAS PRI PLI ARI ALI PRS PLS ARS ALS IPR SPR IAR SAR IPL SPL IAL SAL PIR PSR AIR ASR PIL PSL AIL ASL'.split())
-
-    parser.usage.addSection("\nMulti-component operations on ITK composite warping fields:")
-    parser.add_option(name='-mcs',
-                      description='Multi-component split: Split ITK warping field into three separate displacement fields. The suffix _X, _Y and _Z will be added to the input file name.',
-                      mandatory=False)
-    parser.add_option(name='-omc',
-                      description='Multi-component merge: Merge inputted images into one multi-component image. Requires several inputs.',
-                      mandatory=False)
-
-    parser.usage.addSection("\nWarping field operations:")
-    parser.add_option(name='-display-warp',
-                      description='Create a grid and deform it using provided warping field.',
-                      mandatory=False)
-
-    parser.usage.addSection("\nMisc")
-    parser.add_option(name="-v",
-                      type_value="multiple_choice",
-                      description="""Verbose. 0: nothing. 1: basic. 2: extended.""",
-                      mandatory=False,
-                      default_value=param_default.verbose,
-                      example=['0', '1', '2'])
+
+    parser = argparse.ArgumentParser(
+        description='Perform manipulations on images (e.g., pad, change space, split along dimension). '
+                    'Inputs can be a number, a 4d image, or several 3d images separated with ","',
+        formatter_class=SmartFormatter,
+        add_help=None,
+        prog=os.path.basename(__file__).strip(".py"))
+    mandatory = parser.add_argument_group("MANDATORY ARGUMENTS")
+    mandatory.add_argument(
+        "-i",
+        nargs="+",
+        metavar=Metavar.file,
+        help='Input file(s). If several inputs: separate them by a coma without white space. Example: "data.nii.gz"',
```

Comment:
separation should be with space, not comma (see refactoring or previous functions)

Reply:
note: this is exactly the reason we need to refactor this before 4.0.0 (this is where retrocompat breaks)

Reply:
Oops forgot to update the descirption! Changed in commit [831b857](https://github.com/neuropoly/spinalcordtoolbox/commit/831b857b434e67eb948b8cdc82d78fead6c070fe)